setup.sh

```diff
@@ -453,7 +454,9 @@ elif [ $task == "--install-conda" ]; then
         # which may be the case on some hosts.  See discussion at
         # https://github.com/ContinuumIO/anaconda-issues/issues/11154#issuecomment-535571313
         export TMPDIR=/data/$USER/mambaforge
-    fi
+        sed -i 's|export PATH="$PATH:$HOME/mambaforge/condabin"|export PATH="$PATH:~/data/$USER/mambaforge/condabin"|' .path
```

Comment:
should be `/data` rather than `~/data`, but will fix over in #22

Reply:
Ah, and `.path` should be `~/.path`; otherwise it'll make the change to the version of `.path` that's in this repo, rather than the one actually used in the user's home dir. Will also fix in #22.